lightcycle-integration-test/src/main/java/com/soundcloud/lightcycle/sample/basic/callback/ActivityCallback.java

```diff
@@ -0,0 +1,6 @@
+package com.soundcloud.lightcycle.sample.basic.callback;
+
+public enum ActivityCallback {
```

Comment:
This could be in `ActivityLifecycleCallback` because anyways this callbacks are in `ActivityLightCycle` along with the others (`onCreate`, ...). 

I don't feel strong tho.